src/field_effect.c

```diff
@@ -2521,7 +2531,8 @@ static void TeleportWarpInFieldEffect_SpinEnter(struct Task *task)
             objectEvent->triggerGroundEffectsOnMove = TRUE;
             sprite->subspriteMode = task->data[14];
         }
-    } else
+    }
+    else
```

Comment:
As above

Reply:
Merged upcoming which includes the pret merge that fixes this